### PR TITLE
Wifi support

### PR DESCRIPTION
**Pull request by psppalash**

### CODE REVIEW

**Review by ryannowarm — APPROVED** [review]

**Comment by psppalash** [comment]
Waiting for https://github.com/armPelionEdge/maestro/pull/117 to be merged.

**Review by costanic — DISMISSED** [review]
Does the maestro SHA need to be bumped to pull in maestro wifi support?

**Comment by psppalash** [comment]
> Does the maestro SHA need to be bumped to pull in maestro wifi support?

Yes, but do we wait for the wifi feature to be merged to master? or do we update the SHA to point to latest SHA in maestro dev branch?

**Comment by costanic** [comment]
> > Does the maestro SHA need to be bumped to pull in maestro wifi support?
> 
> Yes, but do we wait for the wifi feature to be merged to master? or do we update the SHA to point to latest SHA in maestro dev branch?

Usually when our team merges dev->master, we won't rebase which means the SHA won't change.  So in this case, it's up to you.

File: README.md
Comment (mray190) [inline README.md:133]: Should add this connect option to the connect.sh script

**Review by mray190 — CHANGES_REQUESTED** [review]
Just a small change then this looks good to me.

Developers use `connect.sh` to connect all the snap plugs - need to update that script with the new plug

File: README.md
Comment (psppalash) [reply README.md:133]: Got it. Will make the required change.

**Review by mray190 — APPROVED** [review]
LGTM

**Comment by mray190** [comment]
Will merge after v1.18.1 makes it to `master`

**Pull request merged**
